### ISSUE TITLE
[BUG] Command not found after updating

### ISSUE THREAD

**Issue by Josee9988** [npm/cli#3000]
### Current Behavior:

I updated npm to the latest version and then I can not use it anymore it says: Command npm not found

### Steps To Reproduce:

sudo npm update -g npm@latest

### Environment:
- OS: Ubuntu 20.04
- Node: 14.16.0
- npm: i don't know because I can not use it

---

This error already happened to me a couple of times. Npm tells me that there is an available update, I install it and then I just can't use npm anymore. So I have to purge node and npm and reinstall both again.

![image](https://user-images.githubusercontent.com/38838976/113051422-3a153880-91a6-11eb-8f09-37a19deb714c.png)

**Comment by darcyclarke** [comment]
@Josee9988 can you reproduce this on the latest version of npm? (ie. install the latest `node` `v16.x` & it should come w/ `npm` `>=v7.13.0` & then try running `sudo npm i -g npm@latest`); Also, how did you install node/npm originally?

**Comment by Josee9988** [comment]
Hi @darcyclarke  I've tried updating it again and it didn't cut the symlink or whatever it was that didn't let me use npm.

Originally I installed it using apt like `sudo apt install nodejs npm`.

As I mentioned this error happened a couple of times but since this issue I've had no problems updating npm,
My actual node version `14.17.0` and my actual npm version (I just updated it to check if it broke again) `7.14.0`.

Thanks for your support

ps: When this error happened I managed to install it again by purging all the node and npm files and reinstalling them again because the update didn't let me use npm.
